CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.7.0-beta.1
+
+## Features ✨ and improvements 🏁
+
+* Enable preloading tiles for camera animation. ([#11328](https://github.com/mapbox/mapbox-gl-js/pull/11328))
+* Add perspective correction for non-rectangular image, canvas and video sources. ([#11292](https://github.com/mapbox/mapbox-gl-js/pull/11292))
```

Comment:
We try to give the changelog some order of importance based on the change. I think this one could be reordered further down as it's relatively minor compared to changes referenced further below. A possible order suggestion (feel free to adjust if you think there's a better one):

- preloading of camera animations
- improved transparency of lines
- improved performance of markers
- improved accessibility
- custom attribution order
- function chaining update to `map.setProjection`

I'd apply similar rearranging to the bug fix list as well.

Reply:
Sorry, I meant to reference the `Add function call chaining to function `map.setProjection`. ([#11279](https://github.com/mapbox/mapbox-gl-js/pull/11279)) (h/t [lpizzinidev](https://github.com/lpizzinidev))` entry. Not perspective correction one!

Reply:
got it!